aidanniceberg/Chat-App
# Set Up Docker Configuration

## Description
Set up docker-compose file, dockerfiles, and connected db to backend

## QA
Run `docker-compose run --rm --service-ports backend` and hit http://localhost/ to see hello world
